src/index.html

```diff
@@ -11,8 +11,52 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute should point to a `.css` file instead of a `.scss` file. Ensure that the SCSS is compiled to CSS and update the link accordingly.